src/AgFEM/CellAggregation.jl

```diff
@@ -53,43 +55,60 @@ function aggregate(
   geo::CSG.Geometry,
   in_or_out)
 
+  trian = Triangulation(cut,geo,in_or_out)
+  cell_to_cut_meas = cell_measure(trian,num_cells(cut.bgmodel))
   cell_to_inoutcut = compute_bgcell_to_inoutcut(cut,geo)
   facet_to_inoutcut = compute_bgfacet_to_inoutcut(cut_facets,geo)
-  aggregate(strategy,cut.bgmodel,in_or_out,cell_to_inoutcut,facet_to_inoutcut)
+  aggregate(strategy,cut.bgmodel,in_or_out,cell_to_cut_meas,cell_to_inoutcut,facet_to_inoutcut)
 end
 
-function aggregate(strategy,model::DiscreteModel,in_or_out,cell_to_inoutcut,facet_to_inoutcut)
+function aggregate(
+  strategy,model::DiscreteModel,in_or_out,cell_to_cut_meas,cell_to_inoutcut,facet_to_inoutcut)
   @abstractmethod
 end
 
-struct AggregateAllCutCells end
+struct AggregateCutCellsByThreshold
+  threshold::Float64
+  AggregateCutCellsByThreshold(x) =
+    ( x < 0.0 ) || ( x > 1.0 ) ? error("Agg. threshold must be in [0,1]") : new(x)
+end
+
+struct AggregateAllCutCells
```

Comment:
Since `AggregateAllCutCells` is not used as struct any more, please, remove the struct and keep only the function
```julia
AggregateAllCutCells() = AggregateCutCellsByThreshold(1.0)
```

Reply:
Done